### PR TITLE
Add -clear-cache flag to src action exec

### PR DESCRIPTION
**Pull request by mrnugget**
This fixes https://github.com/sourcegraph/src-cli/issues/165 and works like this:

Screen Shot 2020-03-30 at 16 25 38

### CODE REVIEW

File: cmd/src/actions_exec.go [hunk 1]
```diff
@@ -161,6 +164,9 @@ Format of the action JSON files:
 			return errors.New("cache is not a valid path")
 		}
 
+		if *clearCacheFlag {
```
Comment (eseliger) [inline cmd/src/actions_exec.go]: Hm?

Comment (mrnugget) [reply cmd/src/actions_exec.go]: Compiler optimization: if I check for the flag, the compiler will more likely put the boolean into a register so the check will be faster down the line.

Comment (mrnugget) [reply cmd/src/actions_exec.go]: (Just kidding of course, I forgot to remove this 😛 )

Comment (eseliger) [reply cmd/src/actions_exec.go]: The world of Go is endless 🙃

**Pull request merged**